src/main/resources/assets/advancedrocketry/lang/de_DE.lang

```diff
@@ -167,7 +167,7 @@ item.pressureTank.0.name=Niedrigdrucktank
 item.pressureTank.1.name=Drucktank
 item.pressureTank.2.name=Hochdrucktank
 item.pressureTank.3.name=Extremer Hochdrucktank
-item.elevatorChip.name=Raumfahrstull Chip
+item.elevatorChip.name=Raumfahrstuhl Chip
```

Comment:
Weltraumlift would be a better name, you don't really say Raumfahrstuhl in German.